### PR TITLE
Add ruling tutorial puzzle for common "Hyper Blaze" interaction

### PR DESCRIPTION
**Pull request by AGoodDoctor**
A tutorial puzzle showing that Hyperblaze works as intended, for people not familar with how cards like those works.

### CODE REVIEW

**Comment by AGoodDoctor** [comment]
Changed :)

File: Tutorials/Tutorial_Ruling_Hyperblaze.lua [hunk 1]
```diff
@@ -0,0 +1,33 @@
+-Created by AGoodDoctor (FTL) using senpaizuri's Puzzle Maker (updated by Naim & Larry126)
+--Partially rewritten by edo9300
+
+Debug.ReloadFieldBegin(DUEL_ATTACK_FIRST_TURN+DUEL_SIMPLE_AI,5)
+Debug.SetPlayerInfo(0,8000,0,0)
+Debug.SetPlayerInfo(1,8000,0,0)
+
+Debug.ShowHint("Special Summon-only monsters need to be Special Summoned by their own condition written on the card to be properly Special Summoned.")
+Debug.ShowHint("If they're summoned ingoring conditions, sent directly to the graveyard/banished, or their summon is negated, you cannot Special Summon them again from elsewhere.")
```
Comment (kevinlul) [inline Tutorials/Tutorial_Ruling_Hyperblaze.lua:9]: Typo in `ignoring`. I suggest wording this as `If their Summon is negated or if they are Summoned ignoring conditions, sent from the hand/Deck to the Graveyard, or banished from the hand/Deck, ...` to clarify what `directly` means and other wordings to be super precise.

File: Tutorials/Tutorial_Ruling_Hyperblaze.lua [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Debug.ShowHint("Summon Uria, then Hamon, then Raviel, then link them all to Decode Talker. Then activate Hyperblaze's effect to special summon a Sacred Beast.")
```
Comment (kevinlul) [inline Tutorials/Tutorial_Ruling_Hyperblaze.lua:10]: For the same reason, `use them to Summon Decode Talker` and capitalize `Special Summon`.

File: Tutorials/Tutorial_Ruling_Hyperblaze.lua [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Debug.ShowHint("Remember that cards such as Chaos or Dark Summoning Beasts do not properly Summon the the Sacred Beasts. Therefore Hyperblaze can't special summon them from the graveyard.")
```
Comment (kevinlul) [inline Tutorials/Tutorial_Ruling_Hyperblaze.lua:11]: For the same reason, capitalize gameplay terminology and spell out the two other cards you are referring to here.

File: Tutorials/Tutorial_Ruling_Hyperblaze.lua [hunk 4]
```diff
@@ -0,0 +1,33 @@
+-Created by AGoodDoctor (FTL) using senpaizuri's Puzzle Maker (updated by Naim & Larry126)
+--Partially rewritten by edo9300
+
+Debug.ReloadFieldBegin(DUEL_ATTACK_FIRST_TURN+DUEL_SIMPLE_AI,5)
+Debug.SetPlayerInfo(0,8000,0,0)
+Debug.SetPlayerInfo(1,8000,0,0)
```
[the remark this answers is not in the document]
Comment (AGoodDoctor) [reply Tutorials/Tutorial_Ruling_Hyperblaze.lua]: That sounds like a pretty cool idea. Also, think I cleaned up everything else. I'm open to more feedback 👍

**Pull request merged**